src/iosanita/contenttypes/restapi/services/modulistica_items/get.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from iosanita.contenttypes.interfaces.cartella_modulistica import (
+    ICartellaModulistica,
+)
+from plone.dexterity.utils import iterSchemata
+from plone.restapi.interfaces import IExpandableElement
+from plone.restapi.interfaces import IFieldSerializer
+from plone.restapi.interfaces import ISerializeToJsonSummary
+from plone.restapi.serializer.converters import json_compatible
+from plone.restapi.services import Service
+from Products.CMFCore.interfaces import IFolderish
+from zope.component import adapter
+from zope.component import queryMultiAdapter
+from zope.interface import implementer
+from zope.interface import Interface
+from zope.schema import getFields
+
+
+@implementer(IExpandableElement)
+@adapter(ICartellaModulistica, Interface)
+class ModulisticaItems(object):
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
+
+    def __call__(self, expand=False):
+        result = {
+            "modulistica-items": {
+                "@id": "{}/@modulistica-items".format(self.context.absolute_url())
+            }
+        }
+        if not expand:
+            return result
+
+        data = self.get_modulistica_data()
+        if data:
+            result["modulistica-items"] = {"items": data}
+        return result
+
+    def get_modulistica_data(self, context=None):
+        if context is None:
+            context = self.context
+        res = []
+        for brain in context.getFolderContents():
+            if brain.portal_type == "Document" and brain.getId == "multimedia":
```

Comment:
Qui usiamo il naming nuovo delle sottocartelle (tipo immagini e documenti).
Se la cartella modulistica deve creare delle sottocartelle di default, allora va messa la configurazione in common.py

Reply:
Ok, aggiorno i nomi, ma non vengono create sottocartelle

Reply:
allora non capisco a cosa serve quel controllo sulla cartella "multimedia" se non viene creata in automatico (parlo anche di iocomune)